xml/release-notes.xml

```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
    </para>
   </sect2>
 
+  <sect2 xml:id="sec.desktop.qt-hidpi">
+   <!-- boo#1091103 -->
+   <title>Using Automatic Scaling in Qt Applications in Setups Which Mix High-DPI/Regular-DPI Monitors</title>
+   <para>
+    Qt supports automatic per-monitor scaling on X. It uses the DPI value of
+    the virtual X screen to calculate the font size for the primary monitor.
+    By default, this value is 96 DPI. It uses the relative DPI of the primary
+    monitor to derive font DPI for all other monitors.
+   </para>
+   <para>
+    Two widely used desktops will override this behavior of Qt, therefore this
+    note does not apply to them:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      GNOME will set DPI according to each monitor's reported scaling.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      KDE Plasma disables automatic scaling by default.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    On other desktops, this behavior of Qt can lead to undesirable situations
+    such as the following: If the primary display is High-DPI
+    (&gt;= 144 DPI), fonts in Qt applications that request scaling, such as
+    VLC, are effectively scaled to half the desired size on all monitors.
```

Comment:
Is this a misunderstading of what 0.5x scale means?

Reply:
No, this is exactly how it appears.